src/components/sectionsWheelPicker/SectionsWheelPicker.driver.tsx

```diff
@@ -1,21 +1,19 @@
-import _ from 'lodash';
+import React from 'react';
 import {useComponentDriver, ComponentProps} from '../../testkit/new/Component.driver';
 import {WheelPickerDriver} from '../WheelPicker/WheelPicker.driver';
 import {SectionsWheelPickerProps} from './index';
 
 export const SectionsWheelPickerDriver = (props: ComponentProps) => {
   const driver = useComponentDriver<SectionsWheelPickerProps>(props);
-  const sections = driver.getProps().children as SectionsWheelPickerProps;
-  const sectionsDrivers = _.map(sections, (_, index) => {
-    const sectionTestID = `${props.testID}.${index}`;
-    return WheelPickerDriver({
-      renderTree: props.renderTree,
-      testID: sectionTestID
-    });
-  });
+  const sectionsDrivers = React.Children.toArray(driver.getElement().props.children).map((section) => {
+    if (typeof section === 'object' && 'props' in section) {
```

Comment:
What is this check 'props' in section?

Reply:
It was needed for TS. React.Children.ToArray return types are unknown are any so It let me use testID and props.
I changed it to be a little more readable because this is kind of weird in second thought.